<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dev_UI</class>
 <widget class="QWidget" name="Dev_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1535</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="Filtered_PPG">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>390</y>
     <width>821</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>Filtered PPG signal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_switch_to_user_ui">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>730</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Switch to User UI</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="Raw_PPG">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>821</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Raw PPG signal</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="Set_RTC">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>631</width>
     <height>601</height>
    </rect>
   </property>
   <property name="title">
    <string>Set RTC</string>
   </property>
   <widget class="QPushButton" name="btn_set_rtc">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>560</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Set time</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbb_mode_time">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>231</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Date time mode 12h</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Date time mode 24h</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Epoch time</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_epoch_time">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Epoch time (GMT0):</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_mode_time">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode time:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_epoch_time">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>520</y>
      <width>291</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_date_time">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Date time (GMT7):</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="te_mode_24">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>430</y>
      <width>191</width>
      <height>26</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>hh:mm:ss</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_set_ddmy">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set day, date, month, year:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_mode_24">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set time (mode 24h):</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_mode_12">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>470</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set time (mode 12h):</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="te_mode_12">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>470</y>
      <width>191</width>
      <height>26</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>hh:mm:ss AP</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Calendar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>431</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Calendar</string>
    </property>
    <widget class="QCalendarWidget" name="calendar_widget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>392</width>
       <height>236</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Notification">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>630</y>
     <width>631</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Error Notification</string>
   </property>
   <widget class="QLabel" name="lbl_err_noti">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>40</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Error Status:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_err_noti">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>451</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
